CHANGES.md

```diff
@@ -45,6 +45,9 @@ Release Notes.
 * Fix `SelfRemoteClient` self observing metrics.
 * Add env variables `SW_CLUSTER_INTERNAL_COM_HOST` and `SW_CLUSTER_INTERNAL_COM_PORT` for cluster selectors `zookeeper`,`consul`,`etcd` and `nacos`.
 * Doc update: `configuration-vocabulary`,`backend-cluster` about env variables `SW_CLUSTER_INTERNAL_COM_HOST` and `SW_CLUSTER_INTERNAL_COM_PORT`.
+* Fix IoTDB Storage Option insert null index value.
+* Set the default value of SW_STORAGE_IOTDB_SESSIONPOOL_SIZE to 2 * CPU_Cores.
```

Comment:
Should be less?  Check PersistenceTimer codes, the default prepare thread is only 2.

```java
prepareExecutorService = Executors.newFixedThreadPool(moduleConfig.getPrepareThreads());
```

How many threads are you building in the bulk metric and segment processing?

Reply:
The default value of SW_STORAGE_IOTDB_SESSIONPOOL_SIZE is 8, while 2 for low-resource e2e.